guides/source/active_record_querying.md

```diff
@@ -815,105 +889,114 @@ SELECT * FROM articles WHERE id > 10 ORDER BY id DESC LIMIT 20
 The `reselect` method overrides an existing select statement. For example:
 
 ```ruby
-Post.select(:title, :body).reselect(:created_at)
+Book.select(:title, :isbn).reselect(:created_at)
 ```
 
 The SQL that would be executed:
 
 ```sql
-SELECT `posts`.`created_at` FROM `posts`
+SELECT `books`.`created_at` FROM `books`
 ```
 
-In case the `reselect` clause is not used,
+Compare this to the case where the `reselect` clause is not used:
 
 ```ruby
-Post.select(:title, :body).select(:created_at)
+Book.select(:title, :isbn).select(:created_at)
 ```
 
 the SQL executed would be:
 
 ```sql
-SELECT `posts`.`title`, `posts`.`body`, `posts`.`created_at` FROM `posts`
+SELECT `books`.`title`, `books`.`isbn`, `books`.`created_at` FROM `books`
 ```
 
 ### `reorder`
 
-The `reorder` method overrides the default scope order. For example:
+The `reorder` method overrides the default scope order. For example if the class definition includes this:
 
 ```ruby
-class Article < ApplicationRecord
-  has_many :comments, -> { order('posted_at DESC') }
+class Author < ApplicationRecord
+  has_many :books, -> { order('year_published DESC') }
```

Comment:
It kind of bugs me that we are using `desc`, `DESC`, `asc`, and `ASC` for orders. We should be more consistent, or opt for hash syntax.